easybuild/easyconfigs/g/GEMMA/GEMMA-0.98.5-foss-2021b.eb

```
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Updated to 0.98.5
+# J Sassmannshausen NHS/GSTT
+
+easyblock = 'MakeCp'
+
+name = 'GEMMA'
+version = '0.98.5'
+
+homepage = 'https://github.com/genetics-statistics/GEMMA'
+description = "Genome-wide Efficient Mixed Model Association"
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/genetics-statistics/GEMMA/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    '3ed336deee29e370f96ec8f1a240f7b62550e57dcd1694245ce7ec8f42241677',
+]
+
+builddependencies = [
+    ('Eigen', '3.4.0'),
+    ('Perl', '5.34.0'),
+    ('Ruby', '3.0.1'),
+]
+
+dependencies = [
+    ('GSL', '2.7'),
+    ('zlib', '1.2.11'),
+    ('FlexiBLAS', '3.0.4'),
+]
+
+buildopts = 'WITH_OPENBLAS="" LIBS="$LIBS $LIBBLAS -lgsl -lz "'  # Needed for FlexiBLAS
+testopts = 'WITH_OPENBLAS="" LIBS="$LIBS $LIBBLAS -lgsl -lz"'  # Needed for the make check below
+
+files_to_copy = ["bin", "doc", "example", "LICENSE", "README.md", "RELEASE-NOTES.md", "scripts", "VERSION"]
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ["bin/gemma"],
+    'dirs': []
+}
+
```

could also add some sanity check commands
```suggestion

sanity_check_commands = ["gemma -h", "gemma -license"]

```

---

I thought the `runtest = 'check'` would have been enough as usually these ones are more thorough than the more simple `binaryname --help` ones we are using. That said, I have added that one as well as that will test the installation, rather than the compilation, I hope.